mlflow/pipelines/steps/train.py

```diff
@@ -642,6 +681,25 @@ def render_schema(inputs, title):
                 f"<b>Best parameters:</b><br>" f"<pre>{best_parameters}</pre><br><br>",
             )
 
+        # Tab 8.1: Best AutoML Parameters
+        if self.step_config["using"].startswith("automl"):
```

Comment:
i think we can collapse this into the `if os.path.exists(best_parameters_yaml):` section above now that we are also writing that file

Reply:
ok, for AutoML, we also output the name and class of the estimator.

Reply:
Done!